File: ProtoScaner.Server/Models/Insidencia.cs
using System;
using System.Collections.Generic;

namespace ProtoScaner.Server.Models;

public partial class Insidencia
{
    public int IdInsidencias { get; set; }

    public int? IdEntregas { get; set; }

    public int? IdPaciente { get; set; }

    public int? IdProtesis { get; set; }

    public int? IdUsuario { get; set; }

    public string? Componentes { get; set; }

    public DateOnly? Fecha { get; set; }

    public string? Descripcion { get; set; }

    public virtual Entrega? IdEntregasNavigation { get; set; }

    public virtual Paciente? IdPacienteNavigation { get; set; }

    public virtual Protesi? IdProtesisNavigation { get; set; }

    public virtual Mantenimiento? IdUsuarioNavigation { get; set; }
}

File: protoscaner.client/src/types/Insidencia.ts
// src/types/Insidencia.ts
export interface Insidencia {
    idInsidencias: number;
    idEntregas?: number;
    idPaciente?: number;
    idProtesis?: number;
    idUsuario?: number;
    componentes?: string;
    fecha?: string; // o Date si tu aplicación maneja bien el tipo Date en lugar de string
    descripcion?: string;
}
